addon/components/file-dropzone/component.js

```diff
@@ -39,15 +39,15 @@ const dragListener = new DragListener();
         {{#if dropzone.supported}}
           Drag and drop images onto this area to upload them or
         {{/if}}
-        {{#file-upload name="photos"
-                      accept="image/*"
-                      multiple=true
-                      onfileadd=(action "uploadImage")}}
+        <FileUpload name="photos"
+                    accept="image/*"
+                    multiple=true
```

Comment:
`multiple=true` is not valid HTML, is it? I think it should be `<FileUpload multiple />`.